Apps/Sandcastle/gallery/3D Tiles Feature Styling.html

```diff
@@ -42,45 +42,41 @@
 });
 
 // Load the NYC buildings tileset.
-var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(3839) });
+var tileset = new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(4258) });
 viewer.scene.primitives.add(tileset);
 
 // Color buildings based on their height.
 function colorByHeight() {
     tileset.style = new Cesium.Cesium3DTileStyle({
         color: {
             conditions: [
-                ["${height} >= 300", "rgba(45, 0, 75, 0.5)"],
-                ["${height} >= 200", "rgb(102, 71, 151)"],
-                ["${height} >= 100", "rgb(170, 162, 204)"],
-                ["${height} >= 50", "rgb(224, 226, 238)"],
-                ["${height} >= 25", "rgb(252, 230, 200)"],
-                ["${height} >= 10", "rgb(248, 176, 87)"],
-                ["${height} >= 5", "rgb(198, 106, 11)"],
+                ["Number(${height}) >= 300", "rgba(45, 0, 75, 0.5)"],
+                ["Number(${height}) >= 200", "rgb(102, 71, 151)"],
+                ["Number(${height}) >= 100", "rgb(170, 162, 204)"],
+                ["Number(${height}) >= 50", "rgb(224, 226, 238)"],
+                ["Number(${height}) >= 25", "rgb(252, 230, 200)"],
+                ["Number(${height}) >= 10", "rgb(248, 176, 87)"],
+                ["Number(${height}) >= 5", "rgb(198, 106, 11)"],
                 ["true", "rgb(127, 59, 8)"]
             ]
         }
     });
 }
 
-// Color buildings by their total area.
-function colorByArea() {
-    tileset.style = new Cesium.Cesium3DTileStyle({
-        color: "mix(color('yellow'), color('red'), min(${area} / 10000.0, 1.0))"
-    });
-}
-
 // Color buildings by their latitude coordinate.
 function colorByLatitude() {
     tileset.style = new Cesium.Cesium3DTileStyle({
+        defines: {
+            latitudeRadians: "Number(${latitude}) * 0.0174532925"
```

Comment:
Use built in `radians` function: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling#radians